parsec/core/gui/new_version.py

```diff
@@ -74,8 +74,12 @@ def _fetch_json_releases():
                     continue
                 if release["prerelease"] and not check_pre:
                     continue
+                if sys.platform == "darwin":
```

Comment:
This would be better moved into line 61 (if current_arch ... set win_version):

```python
if sys.platform == "win32" and current_arch == "x86_64":
    installer_suffix = "-win64-setup.exe"
elif sys.platform == "win32" and current_arch == "i386":
    installer_suffix = "-win32-setup.exe"
elif sys.platform == "darwin":
    installer_suffix = "-macos-amd64.dmg"
else:
    # Generic fallback
    return latest_from_head, url
[...]
```